src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int num;
+
+    public Ball(String color,int num) {
+        this.color = color;
+        this.num = num;
+    }
+
+    @Override
+    public String toString() {
+        return "Color: " + getColor() + " \n" + "Number: " + getNum() + "\n";
+    }

```

Comment:
move toString() method after getters

Reply:
Okay, but why? Is it just a convention? It's just for myself.